discord-bot/main.py

```diff
@@ -11,24 +12,49 @@ def load_discord_token():
     except KeyError:
         raise NameError("DISCORD_TOKEN env does not exist. Unable to proceed.")
 
+MY_GUILD = discord.Object(id=os.environ["GUILD_ID"])
```

Comment:
If I have to predict the future, I imagine the .env values will be updated with more values in the future. It will act similar to a configuration file. The `load_discord_token()` will be renamed to `load_environment_variables()` to make the variables more modular.

Reply:
I resolved it by moving the function to `utility.py`. It now supports a default value and if a value is required. A bit more flexible.